redwood-treehouse-lazylayout-view/src/main/kotlin/app/cash/redwood/treehouse/lazylayout/view/ViewLazyList.kt

```diff
@@ -118,7 +118,8 @@ internal class ViewLazyList<A : AppService>(
             source = { interval.itemProvider.get(indexInInterval) },
           )
           val keyAndContent = KeyAndContent(
-            key = interval.keys[indexInInterval],
+            // coalesce the key to the item index in the list
```

Comment:
We subscribe to the religion of [comments should be sentences](https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html).
```suggestion
            // Coalesce the key to the item index in the list.
```